screens/OrdersScreen.js

```diff
@@ -130,109 +125,121 @@ class OrdersScreen extends React.Component {
 
     if (isLoading) {
       return (
-        <LoadingScreen/>
+        <LoadingScreen />
       )
     } else if (haveData) {
       return (
-        <View style={[styles.fullWidthScreen]}>
-          <NavigationEvents
-            onWillFocus={() => {
-              const shiftId = this.props.navigation.getParam('shiftId')
-              const dateRangeToUse = shiftId != null ? 'SHIFT' : this.state.searchFilter.dateRange
+        <ScrollView>
```

Comment:
You cannot use scrollview here. We want just the scroll on FlatList. If you add this, it will cause the scrolltooffset to stop working properly.